src/Framework/BootstrapFramework.php

```diff
@@ -13,6 +13,15 @@ class BootstrapFramework implements Framework
         'print' => 'print',
     ];
 
+    protected $spacings = [
+        '0'  => '0',
+        '1'  => '1',
+        '2'  => '2',
+        '3'  => '4',
+        '4'  => '6',
+        '5'  => '12',
```

Comment:
5 => 12?

Reply:
Thank you for all the work you've already put in!

I believe that is the correct conversion, no?

Tailwind uses 0.25rem steps (https://tailwindcss.com/docs/padding/)
whereas Bootstrap multiplies $spacer, which defaults to 1rem, by .25, .5, 1, 1.5, **3** (https://getbootstrap.com/docs/4.0/utilities/spacing/).

12 * .25rem = 3rem